tasks/Conformity Game.md

```diff
@@ -0,0 +1,70 @@
+# Summary
+To test predictions that social influence in a two-dimensional social geometry results in consolidation, clustering, and continuing diversity, the Conformity Game, which rewarded individuals for adopting the same position as the majority in their group thus motivating individuals to change and maximizing social influence, was constructed.
+
+# References
+Paper: https://psycnet.apa.org/fulltext/1996-01769-008.html#s7
```

Comment:
Need to add a citation in here

Reply:
Latané, Bibb, and Todd L'Herrou. "Spatial clustering in the conformity game: Dynamic social impact in electronic groups." Journal of personality and social psychology 70.6 (1996): 1218.